### PR TITLE
Include praw.ini in repo by default

### PR DESCRIPTION
**Pull request by sunshine-cid**
Including a praw.ini by default will be helpful for new users

### CODE REVIEW

File: praw.ini [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+reddit_url=https://www.reddit.com
+
+# The URL prefix for short URLs.
+short_url=https://redd.it
+
+# The timeout for requests to Reddit in number of seconds
+timeout=16
+
+[bot1]
+username=""
```
Comment (SiddheshDongare) [inline praw.ini]: Add a small description or placeholder text.

File: praw.ini [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+client_id=""
```
Comment (SiddheshDongare) [inline praw.ini]: Add a small description or placeholder text.

File: praw.ini [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+client_secret=""
```
Comment (SiddheshDongare) [inline praw.ini]: Add a small description or placeholder text.

File: praw.ini [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+#refresh_token=""
```
Comment (SiddheshDongare) [inline praw.ini]: Add a password field as well.

**Review by SiddheshDongare — CHANGES_REQUESTED** [review]
Change name from praw.ini to example.ini, which would get added in the README as well.

**Comment by sunshine-cid** [comment]
Changes have been made. Please review

**Pull request merged**